presidio-image-anonymizer/tests/integration/test_image_analyzer_engine_integration.py

```diff
@@ -0,0 +1,28 @@
+from methods import get_image
+from presidio_image_anonymizer import ImageAnalyzerEngine
+from presidio_image_anonymizer.entities import ImageRecognizerResult
+
+
+def test_given_image_then_text_entities_are_recognized_correctly():
+    image = get_image("ocr_test.png")
+    analyzer_results = ImageAnalyzerEngine().analyze(image)
+    assert len(analyzer_results) == 7
+    results = get_results()
+    for i in range(7):
+        assert analyzer_results[i] == results[i]
+
+
+def test_given_image_without_text_then_no_entities_recognized():
+    image = get_image("no_ocr.jpg")
+    analyzer_results = ImageAnalyzerEngine().analyze(image)
+    assert len(analyzer_results) == 0
+
+
+def get_results():
```

Comment:
would be great to have some context to all this information - add some comments explaining what get_results is returning. 
also should get_results be private?

Reply:
Done. hope it's clear enough now :)